addons/payment_payulatam/tests/test_payulatam.py

```diff
@@ -144,3 +225,53 @@ def test_feedback_processing(self):
         self.env['payment.transaction']._handle_feedback_data('payulatam', payulatam_post_data)
         self.assertEqual(tx.state, 'done', 'Payulatam: wrong state after receiving a valid pending notification')
         self.assertEqual(tx.acquirer_reference, 'b232989a-4aa8-42d1-bace-153236eee791', 'Payulatam: wrong txn_id after receiving a valid pending notification')
+
+    @mute_logger('odoo.addons.payment_payulatam.controllers.main')
+    def test_confirmation_webhook_approved(self):
+        tx = self.create_transaction(flow='redirect')
+        self.assertEqual(tx.state, 'draft')
+
+        res = self.url_open(PayuLatamController._confirmation_url, data=self.payulatam_post_confirmation_approved_data)
```

Comment:
We live in a cruel world where a PR changing a lot of things here https://github.com/odoo/odoo/blob/master/addons/payment/tests/http_common.py just got merged. You can use these functions instead of `url_open` =]